worker/deb.go

```diff
@@ -66,14 +67,30 @@ func InstallDebianPackage(ctx context.Context, client *http.Client, ghClient *gi
 	command := []string{"sh", "-c", "dpkg -i " + f.Name() + " && rm " + f.Name()}
 	if background {
 		command = append([]string{"systemd-run", "-q", "--wait"}, command...)
+		if envValues != nil {
+			envs := make([]string, 0, len(envValues)*2)
+			for k, v := range envValues {
+				envs = append(envs, []string{"-p", fmt.Sprintf("Environment=%s=%s", k, v)}...)
+			}
+			command = append(command, envs...)
+		}
 	}
```

Comment:
I think it's better to place argument of `systemd-run`  before command of `systemd-run`

```suggestion
	command := []string{"sh", "-c", "dpkg -i " + f.Name() + " && rm " + f.Name()}
	if background {
		systemdCommand := []string{"systemd-run", "-q", "--wait"}
		if envValues != nil {
			for k, v := range envValues {
				systemdCommand = append(systemdCommand []string{"-p", fmt.Sprintf("Environment=%s=%s", k, v)}...)
			}
			command := append(systemdCommand, command...)
		}
	}

```